src/ui_options.c

```diff
@@ -0,0 +1,197 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include "ui_options.h"
+
+#include <curses.h>
+
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "app.h"
+#include "boot_data.h"
+#include "ui_screen.h"
+#include "utils.h"
+
+static char *format_option_item(struct option *option)
+{
+	const struct option_def *option_def = &OPTIONS[option->id];
+
+	char *value = NULL;
+	char *line = NULL;
+
+	switch (option_def->type) {
+		case OPT_TYPE_BOOLEAN:
+			value = strdup(option->value ? "on" : "off");
+			break;
+		case OPT_TYPE_TOGGLE:
+			value = strdup(option->value ? "first" : "second");
```

Comment:
```suggestion
			value = strdup(option->value ? "UART" : "GPIO");
```

I can see this one is displayed for the UARTC and UARTD. Could we change it like this?

Reply:
Allowed options to customize these labels.